share/completions/choice.fish

```diff
@@ -0,0 +1,7 @@
+complete --command choice --no-files --arguments '/c\t"Specify the list of choices to be created"
+/n\t"Hide the list of choices"
+/cs\t"Specify that the choices are case-sensitive"
+/t\t"Specify the number of seconds"
+/d\t"Specify the default choice"
+/m\t"Specify a message"
+/?\t"Show help"'
```

Comment:
This can be rewritten (and I think is more readable) as:

```fish
complete --command choice --no-files --arguments '/c' --description "Specify the list of choices to be created"
complete --command choice --no-files --arguments '/n' --description "Hide the list of choices"
complete --command choice --no-files --arguments '/cs' --description "Specify that the choices are case-sensitive"
complete --command choice --no-files --arguments '/t' --description "Specify the number of seconds"
complete --command choice --no-files --arguments '/d' --description "Specify the default choice"
complete --command choice --no-files --arguments '/m' --description "Specify a message"
complete --command choice --no-files --arguments '/?' --description "Show help"
```